ci/jenkins/mellanox/scripts/start_ci.sh

```diff
@@ -94,9 +94,16 @@ EOF
     if [[ -z "${ANTREA_CNI_PR}${ANTREA_CNI_BRANCH}" ]];then
         change_image_name $ANTREA_CNI_HARBOR_IMAGE antrea/antrea-ubuntu
```

Comment:
since we use `docker build -t antrea/antrea-ubuntu:latest` above, isn't this already the name of the image? am I missing something?

Reply:
That is true in cases of building the image (either from a branch or from a PR), but in case of pulling the harbor image (if the scripts were run without specifying either a branch or a PR), the image name is different.

Because we wanted to avoid renaming the image name in the antrea.yml file, we opted with changing the harbor image name.

now that we use the `generate-manifests.sh` to generate the antrea.yml with a custom image, we can have a unified name.

I did that in the last update and made the harbor image name the unified name.